EN_README.md

```diff
@@ -0,0 +1,62 @@
+# Study Reminder
```

Comment:
Hi Patrick! Thank you for your contribution. Let's change this EN_README.md to a README.md to make it easier for the whole community to contribute. We can change the README.md to be ES_README.md. 
Let's add a text in the README.md like "Spanish version [here](readme.md)/Versión en español [acá](readme.md)"
What do you think?